README.md

```diff
@@ -413,41 +413,45 @@ You can now make changes in the repository.
 
 ### Building the AWS JDBC Driver for MySQL
 
-To build the AWS JDBC Driver without running the tests, navigate into the aws-mysql-jdbc directory and run the following command:
+To build the AWS JDBC Driver, navigate into the aws-mysql-jdbc directory and run the following command:
 
 ```bash
-gradlew build -x test
-```
-
-To build the driver and run the tests, you must first install [Docker](https://docs.docker.com/get-docker/).  After installing Docker, execute the following commands to create the Docker servers that the tests will run against:
-
-```bash
-$ cd aws-mysql-jdbc/docker
-$ docker-compose up -d
-$ cd ../
+gradlew build
 ```
 
-Then, to build the driver, run the following command:
+To build the driver without running the tests, run the following command:
 
 ```bash
-gradlew build
+gradlew build -x test
 ```
 
 ### Running the Tests
 
-After building the driver, and installing and configuring Docker, you can run the tests in the ```aws-mysql-jdbc``` directory with the following command:
+To run the tests, you must first install [Docker](https://docs.docker.com/get-docker/). After building the driver, and installing and configuring Docker, you can run the tests in the ```aws-mysql-jdbc``` directory with the following command:
 
 ```bash
 gradlew test
 ```
 
-To shut down the Docker servers when you have finished testing:
+To run only the community tests, use the following command:
```

Comment:
Do we need to specify what we mean by community tests? @sergiyv-bitquill @ColinKYuen  what do you think ?

Reply:
Community tests isn't the best name. Base driver functionality tests?